packages/emotion/src/index.js

```diff
@@ -120,6 +120,7 @@ function handleInterpolation(
   if (
     interpolation === undefined ||
     interpolation === null ||
+    interpolation === false ||
     typeof value === 'boolean'
```

Comment:
I think this should be `typeof interpolation === 'boolean'`.

Reply:
@tkh44 The `typeof value === 'boolean'` can also be removed, I must've wrote value instead of interpolation by mistake.